src/callbacks_step/analysis_dg3d.jl

```diff
@@ -220,7 +236,7 @@ function analyze(::Val{:l2_divb}, du, u, t,
     for l in eachnode(dg)
       divb += ( derivative_matrix[i, l] * u[6, l, j, k, element] +
                 derivative_matrix[j, l] * u[7, i, l, k, element] +
-                derivative_matrix[k, l] * u[7, i, j, l, element] )
+                derivative_matrix[k, l] * u[8, i, j, l, element] )
```

Comment:
:cold_sweat:

Reply:
Yeah, I had the same thought when I saw this bug 😬 .